pandas/io/pickle.py

```diff
@@ -32,12 +44,21 @@ def read_pickle(path):
     ----------
     path : string
```

Comment:
the original api dosen't support URL reading, this feature can be added in future.